native/dll/DShowCapture-2/GraphManager.cpp

```diff
@@ -133,15 +137,26 @@ JNIEXPORT jlong JNICALL Java_sage_DShowCaptureDevice_initGraph0
 		rvs[1] = CreateDshowCaptureInfo( env, videoCaptureFilterName, capDevNum, deviceCaps, tvType, 
 			                                 TuningMode, Country );
 
+		strncpy(TuningMode, GetBDAType(deviceCaps, 2), sizeof(TuningMode));
+		slog((env, "Create first DshowCapture %s.\r\n", TuningMode));
+		rvs[0] = CreateDshowCaptureInfo(env, videoCaptureFilterName, capDevNum, deviceCaps, tvType,
```

Comment:
Shouldn't this be rvs[2]? And then for the next block, rvs[3]? (comments also need updating from the copy/paste) this looks like it'd probably be better in a for loop from 0-3.

Reply:
Yes,  missed that.  Actually I see a few other things I need to tidy up in there

Reply:
I'll need to do some reading on how to do this as a loop